src/backend/marsha/core/tests/test_models_liveregistration.py

```diff
@@ -70,59 +65,188 @@ def test_models_liveregistration_fields_lti_id_unique(self):
                 LiveRegistrationFactory(
                     email="sophie@test-fun-mooc.fr",
                     lti_user_id="56255f3807599c377bf0e5bf072359fd",
+                    lti_id="Maths 01",
                     consumer_site=video.playlist.consumer_site,
                     video=video,
                 )
-                self.assertEqual(
-                    context.exception.messages,
-                    {"error": "IntegrityError"},
-                )
+        assert (
+            "duplicate key value violates unique "
+            'constraint "liveregistration_unique_video_lti_idx"'
+            in str(context.exception)
+        )
```

Comment:
I didn't notice it before but why did you change for `assert` in this file ? We use `self.assert*` in all other tests

Reply:
self.assertIn is nicer